Otec-iOS/Otec/LoginViewController.swift

```diff
@@ -13,63 +13,41 @@ class LoginViewController: UIViewController {
     @IBOutlet weak var newspaperTextfield: UITextField!
     override func viewDidLoad() {
         super.viewDidLoad()
-//        
-//        id = dictionary["id"],
-//        title = dictionary["title"],
-//        body = dictionary["body"],
-//        newspaper = dictionary["newspaper_name"],
-//        createdAt = News.creationDateFromDateString(dictionary["created_at"])
-//
-//        let newDictionary = ["id":"ABCDE12345",
-//                             "title":"Titulo",
-//                             "body":"Este es el contenido de la noticia. Ampliaremos.",
-//                             "newspaper_name":"Daily Planet",
-//                             "created_at":"19-12-1989"]
-//        
-//        guard let new = News.init(dictionary:newDictionary) else {
-//            print ("failure creating new")
-//            return
-//        }
-//        let future3 = NewsRepository().findAll()
-        
-        
         
-//        let future2 = NewsRepository().findAll()
-        
-        // Do any additional setup after loading the view.
+        self.checkIfLoggedInThenShowFeed()
     }
 
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+    private func checkIfLoggedInThenShowFeed() {
+        if (NSUserDefaults.standardUserDefaults().objectForKey(Constants.newspaperUserDefaultsKey) != nil) {
+            self.pushFeedsViewController(animated: false)
+        }
+    }
+    
+    private func pushFeedsViewController(animated animated: Bool) {
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let viewController = storyboard.instantiateViewControllerWithIdentifier("NewsFeedViewController") as! NewsFeedViewController
+        self.navigationController?.pushViewController(viewController, animated: animated)
     }
     
     @IBAction func login(sender: AnyObject) {
-        let newspaperDictionary = ["name":newspaperTextfield.text!,
-                                   "description":"El diario de Clark Kent"]
+        let newspaperName = newspaperTextfield.text!
         
-        let newspaper = Newspaper.init(dictionary:newspaperDictionary)
-        let future = NewspaperRepository().findByID(newspaper!.id)
+        let future = NewspaperRepository().findByID(newspaperName)
         future.start() { result in
             switch result {
-            case .Success(let new):
-                print ("newspaper \(new.id)")
-            // You've got your users fetched in this array!
+            case .Success(let newspaper):
+                dispatch_async(dispatch_get_main_queue()) {
+                    NSUserDefaults.standardUserDefaults().setObject(newspaper.id, forKey:Constants.newspaperUserDefaultsKey)
```

Comment:
Same as above here, I would rather create a configuration `struct` or `class` that handles this logic and exposes an API that is more understandable for the `UIViewController` layer.